### PR TITLE
[stable/chaoskube] Feature/structured log flag

### PR DESCRIPTION
**Pull request by syedimam0012**
#### What this PR does / why we need it:
This feature adds a new flag `logFormat` that can be set to either `"text"` (i.e. default) or `"json"` without affecting the existing behaviour.

#### Special notes for your reviewer:
PR in the [source](https://github.com/linki/chaoskube/pull/120) has to be merged before this one, which will return a new `appVersion` that requires to be updated here.  Chart `version`also needs to be bumped.
@linki 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by linki** [comment]
@syedimam0012 Thank you!

* Please leave out all the changed files that are not under `./stable/chaoskube`.
* Bump the chart version to: `0.15.0`
* Bump the `appVersion` and `imageTag` to: `0.13.0` (image will be pushed once we merge your PR)

File: stable/chaoskube/values.yaml [hunk 1]
```diff
@@ -40,6 +40,9 @@ excludedDaysOfYear:
 # Set specific Timezone for Actions to take place
 timezone: UTC
 
+# If nothing set, defaults to "text". Switch to "json" to enable structured logging
+logFormat:
```
Comment (linki) [inline stable/chaoskube/values.yaml:44]: You could use `logFormat: text` here and leave out the `{{- if .Values.logFormat }}` stuff in `deployment.yaml`.

I tend to like that more in this chart but ultimately you can decide 😄

Comment (syedimam0012) [reply stable/chaoskube/values.yaml:44]: That's what I did initially before I had encountered  `logFormat`: `empty` was failing. Happy to revert this and leave "text" in the `helm values`. ☺️

Comment (linki) [reply stable/chaoskube/values.yaml:44]: For upgrading it's actually better to do the `if` stuff. Thank you 😃

**Comment by marc-sensenich** [comment]
@linki would you be able to `/approve` this for it to be released?

File: stable/chaoskube/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: chaoskube
-version: 0.14.0
-appVersion: 0.12.1
+version: 0.15.0
```
Comment (marc-sensenich) [inline stable/chaoskube/Chart.yaml]: With #11780 this version number should change to `1.1.0`

Comment (syedimam0012) [reply stable/chaoskube/Chart.yaml]: Thanks, @marc-sensenich for the update.
@linki would you want me to update this to `1.1.0` ?

**Comment by maorfr** [comment]
/assign
/ok-to-test

@syedimam0012, is this good to go?
if so, would you mind rebasing?

**Comment by syedimam0012** [comment]
> /assign
> /ok-to-test
> 
> @syedimam0012, is this good to go?
> if so, would you mind rebasing?

I think so @maorfr 
I've just rebased again.

**Comment by linki** [comment]
@syedimam0012 @marc-sensenich @maorfr Thank you!

**Pull request merged**